src/main/java/core/basesyntax/ProcessData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface ProcessData {

```

Comment:
don't name interface/classes as verb phrases, use nouns, or sometimes adjectives/Participles for Interfaces